packages/snap/src/index.ts

```diff
@@ -55,7 +53,7 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ origin, request }) =>
           content: panel([
             text('Please approve the following transaction'),
             text(`to: ${tx.to}`),
-            text(`value: ${tx.value} wei`)
+            text(`value: ${BigInt(tx.value?._hex)} wei`)
```

Comment:
will this display the value as hex?  if so as base 10 would be more readable. 

Also what happens when value is undefined?

Reply:
+1 Could you clarify what this line change is trying to accomplish? I think value was supposed to be in the proper format already IIRC

Reply:
without this change, you get `value: [object Object] wei` something wonky in the way we  transafrom the request.tx that causes the value to be ill formed. We have a better [task](https://github.com/celo-org/celo-monorepo/issues/10420) for cleaning up this approve screen dynamically 

<img width="359" alt="image" src="https://github.com/celo-org/gas-snap/assets/8665738/411a0920-930e-49d2-8fee-b8a5feb4fcb9">